### PR TITLE
Added Danish Translation

### PR DESCRIPTION
**Pull request by teodormoq**

### CODE REVIEW

File: i18n/da_DK.json [hunk 1]
```diff
@@ -0,0 +1,253 @@
+{
+    "%s - Picture #%s": "%s - Billed(er) #%s",
+    "%s Item(s) In Cart"": "%s Produkt(er) i kurven",
```
Comment (Danpiel) [inline i18n/da_DK.json]: Codacy Issue found: [Unexpected character ('"' (code 34)): was expecting a colon to separate field name and value](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4649008)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4649008&bid=15568009)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4649008&bid=15568009)

File: i18n/da_DK.json [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+    "Personal Information": "Personlig information",
+    "Phone number": "Telefonnummer",
+    "Phone number is invalid!": "Telefonnummeret er ugyldigt!",
+    "Play video": "Afspil video",
+    "Play video %s": "Afspil video %s",
+    "Please select variant first!": "Vælg først variant!",
+    "Please sign in first!": "Log ind først!",
+    "Please, select desireable variant first!": "Vælg først ønsket variant!",
+    "Please, try removing selected filters and try again!": "Prøv at fjerne valgte filtre, og prøv igen!",
+    "Postal code": Postnummer",
```
Comment (Danpiel) [inline i18n/da_DK.json]: Codacy Issue found: [Unrecognized token 'Postnummer': was expecting ('true', 'false' or 'null')](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4649008)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4649008&bid=15568009)

File: i18n/da_DK.json [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+    "State/Province": "Stat/Provins",
+    "Status:": "Status:",
+    "Street": "Gade",
+    "Street address": "Vejnavn",
+    "Submit": "Indsend",
+    "Submit Review": "Indsend anmeldelse",
+    "Subscribe to Newsletter": "Abonner på nyhedsbrev",
+    "Subscription settings successfully updated": "Abonnementsindstillinger er opdateret",
+    "Subtotal:": "Subtotal:",
+    "Subtotal:": "Subtotal:",
```
Comment (Danpiel) [inline i18n/da_DK.json]: Codacy Issue found: [Duplicate field 'Subtotal:'](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4649008)

**Comment by alfredsgenkins** [comment]
Thanks a lot! @all-contributors please add @teodormoq for translations!

**Pull request merged**